README.md

```diff
@@ -600,8 +600,8 @@ export class User {
 
 ```typescript
 import {classToPlain} from "class-transformer";
-let user1 = classToPlain(user, { groups: ["user"] }); // will contain id, name, email and password
-let user2 = classToPlain(user, { groups: ["admin"] }); // will contain id, name and email
+let user1 = classToPlain(user, { groups: ["user"] }); // will contain id, name and email
+let user2 = classToPlain(user, { groups: ["admin"] }); // will contain id, name, email and password

```

Comment:
This is not right. According to the example, the property `password` is in the group `user`. In that case, the original documentation is correct.

In fact, there is an error in the documentation, but it is not covered in this PR: after using `@Exclude()` in the class,, the properties id and name are removed.